### PR TITLE
Experimental Runtime samples

### PR DESCRIPTION
**Pull request by tracyboehrer**

### CODE REVIEW

File: experimental/samples/runtime/dotnet/echo-bot/echo-bot/echo-bot.csproj [hunk 1]
```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project Sdk="Microsoft.NET.Sdk.Web">
+  <PropertyGroup>
+    <TargetFramework>netcoreapp3.1</TargetFramework>
+    <AspNetCoreHostingModel>OutOfProcess</AspNetCoreHostingModel>
+  </PropertyGroup>
+  <ItemGroup>
+    <PackageReference Include="Microsoft.AspNetCore.Mvc.NewtonsoftJson" Version="3.1.8" />
+    <PackageReference Include="Microsoft.Bot.Builder.Runtime" Version="4.12.0-daily.preview.20201207.192346.a48b032" />
+    <PackageReference Include="MSTest.TestFramework" Version="2.1.2" />
```
Comment (peterinnesmsft) [inline experimental/samples/runtime/dotnet/echo-bot/echo-bot/echo-bot.csproj:10]: I thought I saw the MSTest reference the other day and thought I was going crazy, but I guess not! Do we need this reference? I would assume not?

File: experimental/samples/runtime/dotnet/echo-bot/echo-bot/runtime.json [hunk 2]
```diff
@@ -0,0 +1,17 @@
+{
```
Comment (peterinnesmsft) [inline experimental/samples/runtime/dotnet/echo-bot/echo-bot/runtime.json:1]: We should be able to update this to reflect the latest changes to single adapter, etc.

File: experimental/samples/runtime/dotnet/echo-bot/echo-bot/Dialogs/echo/generated/echo.lu.qna.dialog [hunk 3]
```diff
@@ -0,0 +1,4 @@
+{
+    "$kind": "Microsoft.CrossTrainedRecognizerSet",
+    "recognizers": []
+}
```
Comment (peterinnesmsft) [inline experimental/samples/runtime/dotnet/echo-bot/echo-bot/Dialogs/echo/generated/echo.lu.qna.dialog:4]: Do we expect these samples to be used primarily in Composer? If so, I would remove the generated folder and its contents from both and let it handle producing these directly.

File: experimental/samples/runtime/dotnet/slack-adapter/slack-adapter/Dialogs/echo/generated/echo.lu.qna.dialog [hunk 4]
```diff
@@ -0,0 +1,4 @@
+{
+    "$kind": "Microsoft.CrossTrainedRecognizerSet",
+    "recognizers": []
+}
```
Comment (peterinnesmsft) [inline experimental/samples/runtime/dotnet/slack-adapter/slack-adapter/Dialogs/echo/generated/echo.lu.qna.dialog:4]: Same thing here, if we intend to primarily use in Composer, I would remove this and let Composer handle generating it directly.

**Review by peterinnesmsft — APPROVED** [review]

File: experimental/samples/runtime/dotnet/echo-bot/echo-bot/echo-bot.csproj (hunk 1 above)
Comment (tracyboehrer) [reply experimental/samples/runtime/dotnet/echo-bot/echo-bot/echo-bot.csproj:10]: Probably not for a sample?

File: experimental/samples/runtime/dotnet/echo-bot/echo-bot/runtime.json (hunk 2 above)
Comment (tracyboehrer) [reply experimental/samples/runtime/dotnet/echo-bot/echo-bot/runtime.json:1]: Yes.  This is dependent on an older version of the Runtime package.  The templates and samples will have to change when the newer Runtime package is published.

File: experimental/samples/runtime/dotnet/echo-bot/echo-bot/Dialogs/echo/generated/echo.lu.qna.dialog (hunk 3 above)
Comment (tracyboehrer) [reply experimental/samples/runtime/dotnet/echo-bot/echo-bot/Dialogs/echo/generated/echo.lu.qna.dialog:4]: Another great question.  One avenue could be you could run the sample directly.  The other is that you use it in Composer.  So... Not sure at this point.

File: experimental/samples/runtime/dotnet/echo-bot/echo-bot.sln [hunk 5]
```diff
@@ -0,0 +1,25 @@
+
+Microsoft Visual Studio Solution File, Format Version 12.00
+# Visual Studio Version 16
+VisualStudioVersion = 16.0.30503.244
+MinimumVisualStudioVersion = 10.0.40219.1
+Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "EchoBot", "echo-bot\echo-bot.csproj", "{21547831-73F3-439C-9927-5E7DCA9FD7EA}"
```
[the remark this answers is not in the document]
Comment (tracyboehrer) [reply experimental/samples/runtime/dotnet/echo-bot/echo-bot.sln:6]: P.S.  I was 100% sure you'd ask this.

**Review by ryanlengel — APPROVED** [review]

**Pull request merged**